CHANGELOG.rst

```diff
@@ -7,6 +7,17 @@ The format is based on `Keep a Changelog`_,
 and this project adheres to `Semantic Versioning`_.
 
 
+Version 1.11.3 (2025-02-20)
+---------------------------
+
+* Changed: reduce the timeout for acquiring a lock for a cache folder
+  from infinity to 24h
+* Changed: show a user warning
+  if a lock for a cache folder cannot be acquired in 2s.
+  The warning message suggests how the user can check
+  if the existing lock file needs to be manually deleted
```

Comment:
The warning message suggests how to check
 whether the existing lock file needs to be deleted manually.

Reply:
Sounds good, I updated accordingly.